fenics/Dockerfile

```diff
@@ -14,18 +14,12 @@ RUN conda env update -n base -f ${HOME}/environment.yml && \
 
 USER root
 RUN fix-permissions /home/$DOCKER_USER &&\
-    fix-permissions $CONDA_DIR &&\
-    chmod +x ${HOME}/include_notebooks.sh &&\
-    fix-permissions /home/$DOCKER_USER
-
-USER $DOCKER_UID
-RUN /bin/bash include_notebooks.sh 
-
-USER root
-RUN fix-permissions /home/$DOCKER_USER && \
+    fix-permissions $CONDA_DIR && \
+    chmod +x ${HOME}/include_notebooks.sh && \
+    /bin/bash include_notebooks.sh &&\
+    fix-permissions /home/$DOCKER_USER && \
     chown -R ${DOCKER_USER}:${DOCKER_GID} ${HOME}/*.ipynb
```

Comment:
I thought the `chown` part is already included in the `fix-permissions`.

Reply:
I have not added this line. I copied it from our older files. I think there is no need for it.

Reply:
Ok, then we keep it for now.